terraform/managed-identity.tf

```diff
@@ -0,0 +1,25 @@
+# Managed Identity for the Job App backend
+# This resource enables the application to authenticate with Azure services
+# without storing credentials in configuration files
+
+resource "azurerm_user_assigned_identity" "job_app_backend" {
+  name                = "job-app-backend-identity"
```

Comment:
Not sure If frontend was the same, but would like to see a team name in here and frontend if not already for some clarity